stepactions/gather-cluster-resources/0.1/gather-cluster-resources.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: tekton.dev/v1alpha1
+kind: StepAction
+metadata:
+  name: gather-cluster-resources
+spec:
+  description: >-
+    This StepAction executes the gather extra script that gathers konflux related resources from the cluster.
+    It also allows for a URL with another gather job to be passed to gather custom resources.
+  image: quay.io/konflux-qe-incubator/konflux-qe-tools:latest
+  params:
+    - name: credentials
+      type: string
+      description: A volume containing credentials to the remote cluster
+    - name: kubeconfig
+      type: string
+      description: Relative path to the kubeconfig in the mounted cluster credentials volume
+    - name: ocp-login-command
+      type: string
+      description: Command to log in to the OpenShift cluster
```

Comment:
should we add default value `""` to both of these params - so that users can choose whether they provide kubeconfig or oc login command?

Reply:
fair point, yes, we should.

Reply:
also to the credentials parameter?
It needs to be added together with the kubeconfig (since the kubeconfig is only the path)